src/site/content/en/blog/flexbox-gap/index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Chromium Lands Flexbox `gap`
+subhead: Chromium's CSS Flexbox and Multi-Column layout engines now support gap. `grid-gap` is old news. 
+authors:
+  - adamargyle
+  # - dgrogan
+description: Chromium's CSS Flexbox and Multi-Column layout engines now support gap. `grid-gap` is old news. 
+date: 2020-04-30
+updated: 2020-04-30
+tags:
+  - layout
+  - flexbox
+  - css
+  - gap
+# hero: hero.jpg
+alt: TODO
+---
+
+CSS Grid brought a delightful spacing feature called [`grid-gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap), which quickly became a popular way to put space into layouts. It was easy to fall in love with because it took upon itself so much responsibility with such elegance. Starting in Chromium 85 `grid-gap` is now just [`gap`](https://drafts.csswg.org/css-align-3/#propdef-gap), and you can use it with Flexbox. 💪
+
+## Browser compatibility
+
+At the time of writing `gap` is supported in desktop Firefox (version 63), Firefox for Android (version 63), and Chromium-based browsers (version 85). See [Browser compatibility](https://developer.mozilla.org/docs/Web/CSS/gap#Browser_compatibility) for updates.
+
+```css/3-4/1-2
+.grid {
+  display: grid;
+  grid-gap: 10px;
+  display: flex;
+  gap: 10px;
+}
+```
```

Comment:
This code block seems out-of-place. I think we want it before the `Browser compatibility` section?

![image](https://user-images.githubusercontent.com/4713486/80765414-d0089500-8af7-11ea-9409-5eef8b883ed3.png)

Reply:
done 👍